dm/syncer/syncer.go

```diff
@@ -1618,6 +1618,9 @@ func (s *Syncer) Run(ctx context.Context) (err error) {
 	if err != nil {
 		return err
 	}
+	if !fresh && s.cfg.SafeModeDuration == "0s" {
```

Comment:
maybe we can tolerate 0s, we already have the feature that syncer will try to exit gracefully, without left the outdated checkpoint.

And s.cliArgs.SafeModeDuration should have higher priority